set.mm

```diff
@@ -548850,6 +548900,515 @@ class of symmetric relations as well (like the elements of the class of
+     21-Jul-2021.) $)
+  dftrrels2 $p |- TrRels = { r e. Rels | ( r o. r ) C_ r } $=
+    ( cv cdm crn cxp cin ccom cssr wbr ctrrels ctrs crels df-trrels df-trs wcel
+    wss cvv wb inex1g elv brssr elrels6 biimpi coeq12d sseq12d syl5bb abeqinbi
+    ax-mp wceq ) ABZUJCUJDEZFZULGZULHIZUJUJGZUJPZAJKLMANUNUMULPZUJLOZUPULQOZUNU
+    QRUSAUJUKQSTUMULQUAUHURUMUOULUJURULUJULUJURULUJUIZURUTRAUJQUBTUCZVAUDVAUEUF
+    UG $.
+
+  ${
+    $d r x y z $.
+    $( Alternate definition of the class of transitive relations.  (Contributed
+       by Peter Mazsa, 22-Jul-2021.) $)
+    dftrrels3 $p |- TrRels =
+        { r e. Rels | A. x A. y A. z ( ( x r y /\ y r z ) -> x r z ) } $=
+      ( cv ccom wss wbr wa wi wal crels ctrrels dftrrels2 cotr rabbieq ) DEZQFQ
+      GAEZBEZQHSCEZQHIRTQHJCKBKAKDLMDNABCQOP $.
+  $}
+
+  $( Alternate definition of the transitive relation predicate.  (Contributed
+     by Peter Mazsa, 22-Aug-2021.) $)
+  dftrrel2 $p |- ( TrRel R <-> ( ( R o. R ) C_ R /\ Rel R ) ) $=
+    ( wtrrel cdm crn cxp cin ccom wss wrel df-trrel wceq dfrel6 coeq12d sseq12d
+    wa biimpi pm5.32ri bitri ) ABAACADEFZSGZSHZAIZOAAGZAHZUBOAJUBUAUDUBTUCSAUBS
+    ASAUBSAKALPZUEMUENQR $.
+
+  ${
+    $d R x y z $.
+    $( Alternate definition of the transitive relation predicate.  (Contributed
+       by Peter Mazsa, 22-Aug-2021.) $)
+    dftrrel3 $p |- ( TrRel R <->
+        ( A. x A. y A. z ( ( x R y /\ y R z ) -> x R z ) /\ Rel R ) ) $=
+      ( wtrrel ccom wss wrel wa cv wbr wi wal dftrrel2 cotr anbi1i bitri ) DEDD
+      FDGZDHZIAJZBJZDKUACJZDKITUBDKLCMBMAMZSIDNRUCSABCDOPQ $.
+  $}
+
+  ${
+    $d R r $.
+    $( Element of the class of transitive relations.  (Contributed by Peter
+       Mazsa, 22-Aug-2021.) $)
+    eltrrels2 $p |- ( R e. TrRels <-> ( ( R o. R ) C_ R /\ R e. Rels ) ) $=
+      ( vr cv ccom wss crels ctrrels dftrrels2 wceq coideq id sseq12d rabeqel )
+      BCZNDZNEAADZAEBFGABHNAIZOPNANAJQKLM $.
+  $}
+
+  ${
+    $d R r x y z $.
+    $( Element of the class of transitive relations.  (Contributed by Peter
+       Mazsa, 22-Aug-2021.) $)
+    eltrrels3 $p |- ( R e. TrRels <->
+        ( A. x A. y A. z ( ( x R y /\ y R z ) -> x R z ) /\ R e. Rels ) ) $=
+      ( vr cv wbr wa wi wal ctrrels dftrrels3 wceq breq anbi12d imbi12d 2albidv
+      crels albidv rabeqel ) AFZBFZEFZGZUBCFZUCGZHZUAUEUCGZIZCJBJZAJUAUBDGZUBUE
+      DGZHZUAUEDGZIZCJBJZAJERKDABCELUCDMZUJUPAUQUIUOBCUQUGUMUHUNUQUDUKUFULUAUBU
+      CDNUBUEUCDNOUAUEUCDNPQST $.
+  $}
+
+  $( The element of the class of transitive relations and the transitive
+     relation predicate are the same when ` R ` is a set.  (Contributed by
+     Peter Mazsa, 22-Aug-2021.) $)
+  eltrrelsrel $p |- ( R e. V -> ( R e. TrRels <-> TrRel R ) ) $=
+    ( wcel ccom wss crels wa ctrrels wtrrel elrelsrel anbi2d eltrrels2 dftrrel2
+    wrel 3bitr4g ) ABCZAADAEZAFCZGQANZGAHCAIPRSQABJKALAMO $.
+
+  $( Equality theorem for transitive relation.  (Contributed by Peter Mazsa,
```

Comment:
...for the transitive relation predicate.

Reply:
done